package/kernel/linux/modules/netdevices.mk

```diff
@@ -559,6 +575,25 @@ endef
 $(eval $(call KernelPackage,ixgbe))
 
 
+define KernelPackage/ixgbevf
+  SUBMENU:=$(NETWORK_DEVICES_MENU)
```

Comment:
You add two packages , but the commit title only says you add one. Please extend the commit message